NuGet.config

```diff
@@ -1,15 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <packageRestore>
```

Comment:
If this makes restore implicit, make respective changes in `build.ps1.`

Reply:
Fixed this